samples/language/Arithmetic Operators.qs

```diff
@@ -0,0 +1,87 @@
+/// # Sample
```

Comment:
For file names with multiple words, could you please eliminate the space between them (i.e. `Arithmetic Operators.qs` -> `ArithmeticOperators.qs`)?

Reply:
done